runtime/cmake/caches/aix_ppc-64.cmake

```diff
@@ -21,18 +21,19 @@
 ################################################################################
 
 #TODO: Env vars should be auto detected by platform
+# Prevent CMake from automatically creating export lists for shared libraries
```

Comment:
perhaps leave extra space after the TODO line, which I don't know what it really refers, but like this it looks like it refers to the next 'set' line only
perhaps we can even remove the TODO line!?